k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1161,7 +1174,39 @@ private void processSourceMessage(MessageViewInfo messageViewInfo) {
                     break;
                 }
                 case FORWARD: {
-                    processMessageToForward(messageViewInfo);
+                    try {
+                        processMessageToForward(messageViewInfo, false);
+                    } catch (IOException ioe) {
+                        /**
+                         * Let the user continue composing their message even if we have a problem processing
+                         * the source message. Log it as an error, though.
+                         */
```

Comment:
This sort of comment is fairly redundant. It's fairly obvious from the code what's happening.

Instead of writing this sort of comment, write a test that checks this is what happens.